doc/rose-rug-advanced-tutorials-command-key.html

```diff
@@ -87,7 +87,7 @@ <h2 id="purpose">Purpose</h2>
       particular app.</p>
 
       <p>They work just like the default command for an app but have to be
-      explicitly called by the command scripting of a task.</p>
+      explicitly called by the <code>script</code> of a task.</p>
```

Comment:
I think this needs re-writing along the lines of:

"but have to be explicitly called by the <code>script</code> entry in a task's runtime settings."

Otherwise, this makes it seem like you're saying the task script itself is the only way to handle this (i.e. reads like what's being invoked by script= rather than an added option to the rose task-run entry under script=)